src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -169,6 +169,11 @@ namespace wallet_rpc
       uint32_t account_index;
       std::string label;
 
+      bool stake_transfer;
+      std::string supernode_public_id;
+      std::string supernode_signature;
+      bool allow_low_stake;
+
```

Comment:
Not clear why these fields has been added to `COMMAND_RPC_CREATE_ADDRESS` command?

Reply:
fixed, apologies patch went wrong

Reply:
@Fez29, now I'm getting build error while building your branch:

```
[ 81%] Building CXX object tests/fuzz/CMakeFiles/cold-outputs_fuzz_tests.dir/cold-outputs.cpp.o
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp: In member function ‘bool tools::wallet_rpc_server::on_transfer(const tools::wallet_rpc::COMMAND_RPC_TRANSFER::request&, tools::wallet_rpc::COMMAND_RPC_TRANSFER::response&, epee::json_rpc::error&)’:
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:838:13: error: ‘const struct tools::wallet_rpc::COMMAND_RPC_TRANSFER::request’ has no member named ‘stake_transfer’
     if (req.stake_transfer) {
             ^~~~~~~~~~~~~~
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:840:47: error: ‘const struct tools::wallet_rpc::COMMAND_RPC_TRANSFER::request’ has no member named ‘supernode_signature’
       if (!epee::string_tools::hex_to_pod(req.supernode_signature, supernode_signature))
                                               ^~~~~~~~~~~~~~~~~~~
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:848:47: error: ‘const struct tools::wallet_rpc::COMMAND_RPC_TRANSFER::request’ has no member named ‘supernode_public_id’
       if (!epee::string_tools::hex_to_pod(req.supernode_public_id, W) || !check_key(W))
                                               ^~~~~~~~~~~~~~~~~~~
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:855:134: error: cannot convert ‘bool’ to ‘cryptonote::network_type’ for argument ‘1’ to ‘std::__cxx11::string cryptonote::get_account_address_as_str(cryptonote::network_type, bool, const cryptonote::account_public_address&)’
       std::string supernode_public_address_str = cryptonote::get_account_address_as_str(m_wallet->testnet(), supernode_public_address);
                                                                                                                                      ^
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:856:67: error: ‘const struct tools::wallet_rpc::COMMAND_RPC_TRANSFER::request’ has no member named ‘supernode_public_id’
       std::string data = supernode_public_address_str + ":" + req.supernode_public_id;
                                                                   ^~~~~~~~~~~~~~~~~~~
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:890:62: error: ‘const struct tools::wallet_rpc::COMMAND_RPC_TRANSFER::request’ has no member named ‘allow_low_stake’
       if (amount < config::graft::TIER1_STAKE_AMOUNT && !req.allow_low_stake)
                                                              ^~~~~~~~~~~~~~~
/home/mbg033/dev/graft-project/GraftNetwork/src/wallet/wallet_rpc_server.cpp:897:57: error: ‘const struct tools::wallet_rpc::COMMAND_RPC_TRANSFER::request’ has no member named ‘supernode_public_id’
       if (!add_graft_stake_tx_extra_to_extra(extra, req.supernode_public_id, supernode_public_address, supernode_signature))
                                                         ^~~~~~~~~~~~~~~~~~~
src/wallet/CMakeFiles/wallet_rpc_server.dir/build.make:62: recipe for target 'src/wallet/CMakeFiles/wallet_rpc_server.dir/wallet_rpc_server.cpp.o' failed
```

Reply:
@Fez29, still having build errors. 
Also, not clear what was the intent  - `transfer_split` mentioned in PR title, stake transaction related fields were added to `COMMAND_RPC_TRANSFER_SPLIT` but handled in `on_transfer` - guess 'stake' related fields should be added into `COMMAND_RPC_TRANSFER` as well and handled same way in both `on_transfer` and `on_transfer_split` ?